src/status_im/common/bottom_sheet_screen/view.cljs

```diff
@@ -87,7 +87,7 @@
            [content
             {:insets           insets
              :close            close
-             :scroll-enabled?  @scroll-enabled?
+             :scroll-enabled?  scroll-enabled?
```

Comment:
Bug cause: Early deref rerendered parent and reinitialized states in child components

Reply:
Oh so this was the side effect that i was worried about it, my bad. thank you @Parveshdhull

Reply:
Hmm but passing a ratom is not ideal, no? Then the component unnecessarily needs to be aware that the prop is a ratom

Reply:
We can use some naming convention to clarify that the passed prop is ratom.